tests/unit/utils/test_utils.py

```diff
@@ -878,3 +878,31 @@ def test_parse_duration_hypothesis(duration):
 ])
 def test_mimetype_extension(mimetype, extension):
     assert utils.mimetype_extension(mimetype) == extension
+
+
+class TestCleanupFileContext:
+
+    def test_no_file(self, tmpdir):
+        tmpfile = tmpdir / 'tmp.txt'
+        with utils.cleanup_file(tmpfile):
+            pass
+        assert not os.path.exists(tmpfile)
```

Comment:
```suggestion
        assert not tmpfile.exists()
```

Reply:
Changed, although still keeping `tmpfile = tmp_path / `tmp.txt` or would you prefer to use `tmp_path` directly?